qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/TestBase.java

```diff
@@ -339,16 +335,13 @@ public String createTestLocationTag() {
 	 * Create a Patient in the database and return its Patient Identifier. The
 	 * Patient Identifier is obtained from the database.
 	 *
-	 * @param personUuid
-	 *            The person
-	 * @param patientIdentifierType
-	 *            The type of Patient Identifier to use
-	 * @param source
-	 *            the idgen source to use to generate an identifier
+	 * @param personUuid            The person
+	 * @param patientIdentifierType The type of Patient Identifier to use
+	 * @param source                the idgen source to use to generate an identifier
 	 * @return The Patient Identifier for the newly created patient
 	 */
 	public String createPatient(String personUuid,
-			String patientIdentifierType, String source) {
```

Comment:
the IDE seems to have messed up with the file, could you rectify the issue about how the IDE is indenting out the codes?